server/service/osquery_utils/queries.go

```diff
@@ -890,7 +894,22 @@ SELECT
   '' AS vendor,
   0 AS last_opened_at,
   path AS installed_path
-FROM homebrew_packages;
+FROM homebrew_packages
+WHERE type = 'formula'
+UNION
+SELECT
+  name AS name,
+  version AS version,
+  '' AS bundle_identifier,
+  '' AS extension_id,
+  '' AS browser,
+  'homebrew_packages' AS source,
+  '' AS vendor,
+  0 AS last_opened_at,
+  path AS installed_path
+FROM homebrew_packages
+WHERE type = 'cask'
+AND NOT EXISTS (SELECT 1 FROM file WHERE file.path LIKE CONCAT(homebrew_packages.path, '/%%%%') AND file.path LIKE '%%.app%%' LIMIT 1);
```

Comment:
Double checking, why do we need the `%` at the end?

Reply:
yes as .app bundles contain a trailing `/`.  Could possibly use `file.directory` which doesn't use the trailing `/`.

```
osquery> select path from file where path LIKE '/opt/homebrew/Caskroom/firefox/%%' AND path LIKE '%.app%' LIMIT 1;
 path = /opt/homebrew/Caskroom/firefox/126.0.1/Firefox.app/
```